gui/drawing.tcl

```diff
@@ -153,7 +170,10 @@ proc drawNode { node } {
 		set l [format "%s %s" $l [getIfcIPv4addr $node $ifc]]
 	    }
 	}
-	set label [.panwin.f1.c create text $x $y -fill blue \
+	#set label [.panwin.f1.c create text $x $y -fill $colorNameNode \
```

Comment:
The \ character at the end of the line makes the next line a comment so it is not executed for me. Remove this line completely.